packages/cln_plugin/lib/src/cln_plugin_base.dart

```diff
@@ -1,6 +1,57 @@
 // TODO: Put public facing types in this file.
 
-/// Checks if you are awesome. Spoiler: you are.
-class Awesome {
-  bool get isAwesome => true;
+import 'dart:io';
+
+import 'package:cln_plugin/src/rpcmethods/abstractrpcmethod.dart';
+import 'package:cln_plugin/src/rpcmethods/manifest/getmanifest.dart';
+
+import 'icln_plugin_base.dart';
+
+class Plugin implements ICLNPlugin {
```

Comment:
```suggestion
class Plugin implements CLNPlugin {
```

in java there is a code convention that all the interface need to start by I, in that there is no reason